core/storage/blog/gae_models.py

```diff
@@ -384,7 +386,9 @@ def get_published_models_by_user(
             according to the time when the model was last updated.
         """
         query = cls.query(
-            cls.editor_ids == user_id, cls.blog_post_is_published == True # pylint: disable=singleton-comparison
+            cls.editor_ids == user_id,
+            cls.blog_post_is_published # pylint: disable=singleton-comparison
+            == True
```

Comment:
You cannot move this to the end of the previous line (before the comment)? There is space.

Ditto below.